src/main/java/lotto/constant/LottoRank.java

```diff
@@ -0,0 +1,38 @@
+package lotto.constant;
+
+public enum LottoRank {
+
+    FIRST(6, 2000000000, false),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000, false),
```

이런식으로 언더바를 통해 콤마처럼 구분지을 수 있습니다.